inspector/src/components/popupComponent.tsx

```diff
@@ -72,7 +72,7 @@ export class PopupComponent extends React.Component<IPopupComponentProps, { isCo
             this._window.addEventListener("beforeunload", () => this._window && onClose(this._window));
             this._window.addEventListener("resize", () => {
                 if (this.props.onResize) {
-                    this.props.onResize();
+                    this.props.onResize(this._window!);

```

Comment:
you could check the _window in the previous if to prevent the use of ! which is always scary :-)

Reply:
```suggestion
                    this._window && this.props.onResize(this._window);
```